diff --git a/node_modules/@digitalcredentials/rdf-canonize/lib/MessageDigest-reactnative.js b/node_modules/@digitalcredentials/rdf-canonize/lib/MessageDigest-reactnative.js
index 3d50b27..243abe5 100644
--- a/node_modules/@digitalcredentials/rdf-canonize/lib/MessageDigest-reactnative.js
+++ b/node_modules/@digitalcredentials/rdf-canonize/lib/MessageDigest-reactnative.js
@@ -3,7 +3,7 @@
  */
 'use strict';
 
-const crypto = require('isomorphic-webcrypto');
+// const crypto = require('isomorphic-webcrypto');
 require('fast-text-encoding');
 
 module.exports = class MessageDigest {
@@ -15,7 +15,7 @@ module.exports = class MessageDigest {
   constructor(algorithm) {
     // check if crypto.subtle is available
     // check is here rather than top-level to only fail if class is used
-    if(!(crypto && crypto.subtle)) {
+    if(!(global.crypto && global.crypto.subtle)) {
       throw new Error('crypto.subtle not found.');
     }
     if(algorithm === 'sha256') {
@@ -35,7 +35,7 @@ module.exports = class MessageDigest {
   async digest() {
     const data = new TextEncoder().encode(this._content);
     const buffer = new Uint8Array(
-      await crypto.subtle.digest(this.algorithm, data));
+      await global.crypto.subtle.digest(this.algorithm, data));
     // return digest in hex
     let hex = '';
     for(let i = 0; i < buffer.length; ++i) {
