diff --git a/node_modules/@digitalcredentials/jsonld-signatures/lib/sha256digest-reactnative.js b/node_modules/@digitalcredentials/jsonld-signatures/lib/sha256digest-reactnative.js
index d4012ad..8443cf3 100644
--- a/node_modules/@digitalcredentials/jsonld-signatures/lib/sha256digest-reactnative.js
+++ b/node_modules/@digitalcredentials/jsonld-signatures/lib/sha256digest-reactnative.js
@@ -3,7 +3,7 @@
  */
 'use strict';
 
-const crypto = require('isomorphic-webcrypto');
+// const crypto = require('isomorphic-webcrypto');
 require('fast-text-encoding');
 
 module.exports = {
@@ -17,7 +17,7 @@ module.exports = {
   async sha256digest({string}) {
     const bytes = new TextEncoder().encode(string);
     return new Uint8Array(
-      await crypto.subtle.digest({name: 'SHA-256'}, bytes)
+      await global.crypto.subtle.digest({name: 'SHA-256'}, bytes)
     );
   }
 };
