diff --git a/node_modules/@sphereon/ssi-sdk-ext.did-utils/dist/types.js b/node_modules/@sphereon/ssi-sdk-ext.did-utils/dist/types.js
index 340c49b..1433477 100644
--- a/node_modules/@sphereon/ssi-sdk-ext.did-utils/dist/types.js
+++ b/node_modules/@sphereon/ssi-sdk-ext.did-utils/dist/types.js
@@ -9,6 +9,7 @@ var SupportedDidMethodEnum;
     SupportedDidMethodEnum["DID_ION"] = "ion";
     SupportedDidMethodEnum["DID_EBSI"] = "ebsi";
     SupportedDidMethodEnum["DID_JWK"] = "jwk";
+    SupportedDidMethodEnum["DID_OYD"] = "oyd";
 })(SupportedDidMethodEnum || (exports.SupportedDidMethodEnum = SupportedDidMethodEnum = {}));
 var IdentifierAliasEnum;
 (function (IdentifierAliasEnum) {
diff --git a/node_modules/@sphereon/ssi-sdk-ext.did-utils/src/types.ts b/node_modules/@sphereon/ssi-sdk-ext.did-utils/src/types.ts
index f1e88dd..a6f8d10 100644
--- a/node_modules/@sphereon/ssi-sdk-ext.did-utils/src/types.ts
+++ b/node_modules/@sphereon/ssi-sdk-ext.did-utils/src/types.ts
@@ -10,6 +10,7 @@ export enum SupportedDidMethodEnum {
   DID_ION = 'ion',
   DID_EBSI = 'ebsi',
   DID_JWK = 'jwk',
+  DID_OYD = 'oyd',
 }
 
 export enum IdentifierAliasEnum {
