diff --git a/node_modules/react-native-material-ripple/index.js b/node_modules/react-native-material-ripple/index.js
index 9df9b50..78fe0a4 100644
--- a/node_modules/react-native-material-ripple/index.js
+++ b/node_modules/react-native-material-ripple/index.js
@@ -250,15 +250,18 @@ export default class Ripple extends PureComponent {
       ...('web' !== Platform.OS? { nativeID } : null),
     };

+    const hasRipple = !!this.state.ripples.length
+
     let containerStyle = {
       borderRadius: rippleContainerBorderRadius,
+      zIndex: hasRipple ? 1 : 0
     };

     return (
       <TouchableWithoutFeedback {...touchableProps}>
-        <Animated.View {...props} pointerEvents='box-only'>
+        <Animated.View {...{pointerEvents: "box-only", ...props}}>
           {children}
-          <View style={[styles.container, containerStyle]}>
+          <View pointerEvents="none" style={[styles.container, containerStyle]}>
             {ripples.map(this.renderRipple)}
           </View>
         </Animated.View>